packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -234,16 +233,9 @@ function Thread( {
 					>
 						<CommentBoard
 							thread={ reply }
-							onEdit={
-								'approved' !== thread.status
-									? onEditComment
-									: undefined
-							}
-							onDelete={
-								'approved' !== thread.status
-									? onCommentDelete
-									: undefined
-							}
```

Comment:
Essentially, I wanted to remove the conditions and callback checks within the component.